src/He.PipelineAssessment.UI/Features/Workflow/Views/Shared/_ReviewAnswers.cshtml

```diff
@@ -44,4 +45,12 @@
             }
         </dl>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+@functions {
```

Comment:
We probably want to have these in a single helper function class, as explained [here](https://stackoverflow.com/questions/17602507/using-a-razor-functions-in-several-webpages-cshtml-files) and [here](https://www.mikesdotnetting.com/Article/173/The-Difference-Between-@Helpers-and-@Functions-In-WebMatrix)

Reply:
Created a single helper function as per suggestion.